### ISSUE TITLE
String interpolation filling in value instead of compile time error

### ISSUE THREAD

**Issue by pythonuser00** [rust-lang/rust#93478]
I tried this code:

```rust
fn main() {
    let tup = (500, 6.4, 1);

    let (x, y, z) = tup;

    println!("The value of x, y, z is: {x}, {}, {z}");
}

```

I expected to see this happen: Compile time error for providing an empty parenthesis in `println!("The value of x, y, z is: {x}, {}, {z}");`

Instead, this happened: 

`The value of x, y, z is: 500, 500, 1` The value of variable `x` (`500`) was filled in for the empty parenthesis.

### Meta

`rustc --version --verbose`:
```
<version>
rustc 1.58.1 (db9d1b20b 2022-01-20)                                                                                                                                                 
binary: rustc                                                                                                                                                                       
commit-hash: db9d1b20bba1968c1ec1fc49616d4742c1725b4b                                                                                                                               
commit-date: 2022-01-20                                                                                                                                                             
host: x86_64-pc-windows-msvc                                                                                                                                                        
release: 1.58.1                                                                                                                                                                     
LLVM version: 13.0.0 
```

<details><summary>Backtrace</summary>
<p>

```
<backtrace>
```

</p>
</details>

**Comment by fmease** [comment]
Duplicate of #93378.

**Comment by ehuss** [comment]
Yea, I believe this is a duplicate of #93378 and should be fixed by #93394.
